src/server-glue/module.ts

```diff
@@ -31,6 +31,13 @@ const configLoader = require('../config-loader/config-loader')
 const migrator = require('../migrator/migrator')
 
 export const sbvrUtils = require('../sbvr-api/sbvr-utils')
+export const {
```

Comment:
Can you export these from `sbvr-utils` instead please, since pinejs modules get called with `sbvrUtils` as one of their arguments and so could access the errors directly from there

Reply:
@Page- just to confirm how you intend this to work, consumer projects would do 
```
{ sbvrUtils: { BadRequestError } } = require '@resin/pinejs'
```
?

Or would we prefer `require('@resin/pinejs').sbvrUtils.errors.BadRequestError`?